app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurance/InsuranceDestination.kt

```diff
@@ -145,59 +152,62 @@ private fun InsuranceScreen(
     refreshingOffset = PullRefreshDefaults.RefreshingOffset + systemBarInsetTopDp,
   )
   Box(Modifier.fillMaxSize()) {
-    AnimatedContent(
-      targetState = uiState.isLoading,
-      transitionSpec = {
-        MotionDefaults.fadeThroughEnter togetherWith MotionDefaults.fadeThroughExit
-      },
-      label = "uiState is Loading",
-      modifier = Modifier.fillMaxSize(),
-    ) { isLoading ->
-      when (isLoading) {
-        true -> HedvigFullScreenCenterAlignedProgressDebounced(Modifier.fillMaxSize())
-        false -> {
-          Column(
-            Modifier
-              .fillMaxSize()
-              .pullRefresh(pullRefreshState)
-              .verticalScroll(rememberScrollState())
-              .windowInsetsPadding(WindowInsets.safeDrawing),
-          ) {
-            Spacer(Modifier.height(16.dp))
-            Row(
-              verticalAlignment = Alignment.CenterVertically,
-              modifier = Modifier
-                .height(64.dp)
-                .fillMaxWidth()
-                .padding(horizontal = 16.dp),
-            ) {
-              Text(
-                text = stringResource(id = R.string.DASHBOARD_SCREEN_TITLE),
-                style = MaterialTheme.typography.titleLarge,
-              )
-            }
-
-            if (uiState.hasError) {
-              HedvigErrorSection(onButtonClick = reload)
-            } else {
-              InsuranceScreenContent(
-                imageLoader = imageLoader,
-                contracts = uiState.contracts,
-                crossSells = uiState.crossSells,
-                showNotificationBadge = uiState.showNotificationBadge,
-                onInsuranceCardClick = onInsuranceCardClick,
-                onCrossSellClick = onCrossSellClick,
-                navigateToCancelledInsurances = navigateToCancelledInsurances,
-                quantityOfCancelledInsurances = uiState.quantityOfCancelledInsurances,
-              )
+    Column(
+      Modifier
+        .fillMaxSize()
+        .pullRefresh(pullRefreshState)
+        .verticalScroll(rememberScrollState())
+        .windowInsetsPadding(WindowInsets.safeDrawing),
+    ) {
+      Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+          .height(64.dp)
+          .fillMaxWidth()
+          .padding(horizontal = 16.dp),
+      ) {
+        Text(
+          text = stringResource(id = R.string.DASHBOARD_SCREEN_TITLE),
+          style = MaterialTheme.typography.titleLarge,
+        )
+      }
+      AnimatedContent(
+        targetState = uiState.isLoading,
+        transitionSpec = {
+          fadeIn() togetherWith fadeOut()
+        },
+        label = "uiState is Loading",
+      ) { isLoading ->
+        Column(
+          Modifier
+            .fillMaxSize(),
+        ) {
+          if (uiState.hasError) {
+            HedvigErrorSection(onButtonClick = reload)
+          } else {
```

Comment:
With this there is no animation to and from the error state. Would we perhaps optimally want to have something there too? So that the AnimatedContent was on the uiState in general instead of just the isLoading case?